### ISSUE TITLE
Panic on build on Debian 8

### ISSUE THREAD

**Issue by WillemMali** [moby/moby#32764]
Build script:

```
#!/bin/sh -eu
docker build -t cucumberjs-test .
```

Dockerfile:

```
from node
```

Build log:

```
o~ ./build.sh 
Sending build context to Docker daemon  3.072kB
Step 1/2 : FROM node:wheezy
wheezy: Pulling from library/node

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x8a1dc0]

goroutine 1 [running]:
panic(0xa23fe0, 0xc420012160)
        /usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/docker/docker/vendor/github.com/Nvveen/Gotty.(*TermInfo).GetAttribute(0x0, 0xad1b45, 0x3, 0x0, 
0x100e0c0, 0x10, 0x7f3a8c7ae4b0)                                                                         
        /usr/src/docker/.gopath/src/github.com/docker/docker/vendor/github.com/Nvveen/Gotty/gotty.go:94 +0
x140                                                                                                     
github.com/docker/docker/vendor/github.com/Nvveen/Gotty.(*TermInfo).Parse(0x0, 0xad1b45, 0x3, 0xc4203d4940
, 0x1, 0x1, 0xc42017d2b0, 0x4644e5, 0xc4203fa0c0, 0xc420074200)                                          
        /usr/src/docker/.gopath/src/github.com/docker/docker/vendor/github.com/Nvveen/Gotty/parser.go:42 +
0x5a                                                                                                     
github.com/docker/docker/pkg/jsonmessage.cursorUp(0xfe4000, 0xc4202e2f00, 0xfe4580, 0x0, 0x1)
        /usr/src/docker/.gopath/src/github.com/docker/docker/pkg/jsonmessage/jsonmessage.go:144 +0xec
github.com/docker/docker/pkg/jsonmessage.DisplayJSONMessagesStream(0xfe5100, 0xc4203de240, 0xfe4000, 0xc42
02e2f00, 0x1, 0xc420361701, 0x0, 0x1, 0x0)                                                               
        /usr/src/docker/.gopath/src/github.com/docker/docker/pkg/jsonmessage/jsonmessage.go:263 +0x3e8
github.com/docker/docker/cli/command/image.runBuild(0xc42032b490, 0x7ffc14908366, 0x1, 0x0, 0x0, 0xc420346
c20, 0xb4abe0, 0xc420346c60, 0xb4aca8, 0xc420346c40, ...)                                                
        /usr/src/docker/.gopath/src/github.com/docker/docker/cli/command/image/build.go:296 +0xaae
github.com/docker/docker/cli/command/image.NewBuildCommand.func1(0xc42035c6c0, 0xc4203a4270, 0x1, 0x3, 0x0
, 0x0)                                                                                                   
        /usr/src/docker/.gopath/src/github.com/docker/docker/cli/command/image/build.go:82 +0x9c
github.com/docker/docker/vendor/github.com/spf13/cobra.(*Command).execute(0xc42035c6c0, 0xc42000c110, 0x3,
 0x3, 0xc42035c6c0, 0xc42000c110)                                                                        
        /usr/src/docker/.gopath/src/github.com/docker/docker/vendor/github.com/spf13/cobra/command.go:646 
+0x26d                                                                                                   
github.com/docker/docker/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc42001f440, 0xc42001f8c0, 0xc
420301540, 0xc4200344a8)                                                                                 
        /usr/src/docker/.gopath/src/github.com/docker/docker/vendor/github.com/spf13/cobra/command.go:742 
+0x377                                                                                                   
github.com/docker/docker/vendor/github.com/spf13/cobra.(*Command).Execute(0xc42001f440, 0xc42001f440, 0xfe
54c0)                                                                                                    
        /usr/src/docker/.gopath/src/github.com/docker/docker/vendor/github.com/spf13/cobra/command.go:695 
+0x2b                                                                                                    
main.main()
        /usr/src/docker/.gopath/src/github.com/docker/docker/cmd/docker/docker.go:168 +0xcb
```

I've wiped my Docker image and container caches, so that isn't it. I'm on fully-updated Debian 8, using this Docker version:

    Docker version 17.04.0-ce, build 4845c56

**Comment by WillemMali** [comment]
Apparently this was just a random crapping-out by Docker, the error went away on building a different image in a different build context.

**Comment by WillemMali** [comment]
And it's back again.

**Comment by WillemMali** [comment]
It went away for a while, then came back, and is gone right now. I'm not sure what causes it, but rebooting does seem to help.

**Comment by cpuguy83** [comment]
Thanks! Looks like this got fixed by #32401
